staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -946,6 +946,10 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// Sysctls is a white list of allowed sysctls in a pod spec. Each entry
```

Comment:
Make member name lowercase as it will appear in the api doc: s/Sysctls/sysctls/

Reply:
What is the semantics of empty lists vs. nil in the annotations? Does empty list mean "no sysctls allowed at all" vs. nil "use the default set" ?

Reply:
The original functionality is respected. So
- If the list is empty -> No sysctls are allowed
- If the list is nil -> Use the default sysctls patterns

Reply:
> Make member name lowercase as it will appear in the api doc: s/Sysctls/sysctls/

Reply:
The nil/empty distinction is lost when roundtripping via serialized protobuf, and makes for a fairly confusing API

Reply:
> The nil/empty distinction is lost

Don't we test this in roundtrip tests? We should. 

If we change the API semantics (I don't feel strongly, I tend to agree that it is confusing), we should doc that. Do we want to?